subiquity/common/apidef.py

```diff
@@ -429,6 +430,17 @@ def DELETE(id: str) -> StorageResponseV2:
                     """Delete the Raid specified by its ID. Any associated
                     partition will be deleted as well."""
 
+            class calculate_entropy:
```

Comment:
I would have moved this outside the `v2` namespace but maybe my understanding of this namespace is incorrect.

Reply:
Added to meeting agenda to discuss